### ISSUE TITLE
Not being able to edit the geometry of a geo-object does not allow to adjust temporal validity of such geometry 

### ISSUE THREAD

**Issue by SteeveEbener** [terraframe/geoprism-registry#562]
**List the CGR information**
- CGR version: 0.13.3
- CGR instance (e.g. a URL): Staging

**Describe the bug (clear and concise)**
When the Geo-Object Editing  option is disabled for a given geo-object type (geo-object type and hierarchies module) it is not possible to access the edit geometry window in the geo-object editing modale.

This is fine from the perspective that the RA/RM should not edit the geometry itself but doing so does not allow to modify the temporal validity of such geometry.

**To Reproduce**
As the RA/RM:
1. Go to the lists module
2. View the list for a geo-object under the curation mandate of the RA/RM but for which the geo-object editing is disabled 
3. Click the edit pen button for one of the geo-object 
4. Click the edit button in the geo-object editing modale
5. See the issue: the geometry field is not there and it is not possible to go change the temporal validity for that geometry

**Expected behavior (clear and concise)**
While the RA/RM should not be able to edit the geometry itself, it should be possible to still access the geometry window to adjust its temporal validity.

In other words we need to separate the editing of the geometry from the editing of the temporal validity of such geometry and allow for the later to be adjusted

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Google Chrome
 - Version: The most recent one

**Additional context (if any)**

**Comment by SteeveEbener** [comment]
Tested under 0.14.6 and working. Closing this ticket
